prometheus_client/openmetrics/parser.py

```diff
@@ -141,37 +169,120 @@ def _parse_labels(it, text):
                 break
             else:
                 raise ValueError("Invalid line: " + text)
-    return labels
+        labels_len += 1
+    return labels, labels_len
+
+
+def _parse_labels(text):
+    labels = {}
+    # Return if we don't have valid labels
+    if "=" not in text:
```

Comment:
How would this handle something like `{a}`  ?

Reply:
made the required changes and added some use cases like that